eventing/default-channels.md

```diff
@@ -0,0 +1,129 @@
+# Default Channels
+
+The default channel configuration allows channels to be created without
+specifying a provisioner. This leaves the selection of channel provisioner and
+properties up to the operator. The operator controls the default settings via a
+ConfigMap.
+
+## Creating a default channel
+
+To create a default channel, use the same procedure as for creating a normal
+channel, but leave the `spec.provisioner` property blank. The `spec` property
+must be provided, but should be empty.
+
+_The content of `spec.arguments` will be cleared for default channels._
+
+This is a valid default channel:
+
+```yaml
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Channel
+metadata:
+  name: default-channel
+  namespace: default
+spec: {}
```

Comment:
I think you can omit the curly brackets here and kubectl will do the right thing.

```suggestion
spec:
```

Reply:
I get an error about `/spec/generation` doing that:
```
$ echo "apiVersion: eventing.knative.dev/v1alpha1
                                                      kind: Channel
                                                      metadata:
                                                        name: default-channel
                                                        namespace: default
                                                      spec:" | kubectl apply -f -
Error from server (InternalError): error when creating "STDIN": Internal error occurred: Internal error occurred: jsonpatch add operation does not apply: doc is missing path: /spec/generation
```

I think it's because kubectl is omitting the spec field when the brackets aren't included.